test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/command/queue/QueueCommand.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.eventhub;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.operation;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue.storageQueue;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Util;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.command.queue
+{
+    public class QueueCommand : Command<QueueResponse>, IDisposable
+    {
+        private QueueType queueType;
+        private QueueOperation queueOperation;
+        private string queueName;
+        private IQueueManager<QueueRequest, QueueResponse> queueManager;
+        
+        public QueueCommand(QueueType queueType, QueueOperation queueOperation, string queueName)
+        {
+            this.queueType = queueType;
+            this.queueName = queueName;
+            this.queueOperation = queueOperation;
```

Comment:
is null allowed to pass for the enum in csharp ?
if yes shall we add the check ?

Reply:
Since we are not using a nullable enum we don't need to check for that.